HealthcareSystem/Backend/Service/NotificationSurveyAndFeedback/FeedbackService.cs

```diff
@@ -13,15 +14,22 @@ namespace Service.NotificationSurveyAndFeedback
    public class FeedbackService
    {
         private FeedbackRepository feedbackRepository = new FeedbackRepository();
-      public Model.Users.Feedback AddFeedback(Model.Users.Feedback feedback)
-      {
-            return feedbackRepository.NewFeedback(feedback);
-      }
-      
-      public List<Model.Users.Feedback> ViewFeedbackInformations()
-      {
-            return feedbackRepository.GetAllFeedbacks();
-      }
-   
-   }
+        public void NewFeedback(Feedback feedback)
```

Comment:
I feel like some sort of validation is necessary here, for example that feedback is not null, that the commentator id is valid, and so on.